### PR TITLE
Add stream ID and replay since

### PR DESCRIPTION
**Pull request by cristian-t**

### CODE REVIEW

File: streams/common.proto [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+
 option java_multiple_files = true;
 option java_package = "games.mythical.saga.sdk.proto.streams";
 
 package saga.rpc.streams;
 
 message Subscribe {
     string title_id = 1;
+    string stream_id = 2;
+    oneof replay_since_oneof {
```
Comment (kwang-mythical) [inline streams/common.proto]: what other options could this be?

Comment (cristian-t) [reply streams/common.proto]: this is done to allow it to be nullable, not for options

Comment (kwang-mythical) [reply streams/common.proto]: you could use the keyword optional then

Comment (cristian-t) [reply streams/common.proto]: i've looked into that, it requires newer versions of protobuf 3 and it does the same thing, wraps it in a oneof for you.

**Review by kwang-mythical — APPROVED** [review]

**Pull request merged**